restapi/api.go

```diff
@@ -62,32 +62,22 @@ func Router(db *gorm.DB, locationFinder location.LocationFinder) (http.Handler,
 		rest.GetHandler(r, "/authors/origin_countries", api.ReturnAllCountries)
 		rest.GetHandler(r, "/tags", api.ReturnAllUniqueTags)
 		rest.GetHandler(r, "/login", api.Login)
-		rest.PostHandler(r, "/story", api.CreateStoriesFormData)
-
 		rest.GetHandler(r, "/auth/callback", api.AuthCallback)
 		rest.GetHandler(r, "/client_tokens", api.ReturnClientTokens)
-		rest.GetHandler(r, "/unauthorized", api.UnauthorizedPage)
-
-		// TODO: move these back to private api once https://github.com/uwblueprint/shoe-project/issues/204 is closed
-		rest.PostHandler(r, "/stories", api.CreateStories)
-		rest.PostHandler(r, "/authors", api.CreateAuthors)
+		rest.PostHandler(r, "/story", api.CreateStoriesFormData)
```

Comment:
we probably want this to be protected as well

Reply:
woops, good catch. ty